server/utils/auth/github_auth_router.py

```
@@ -0,0 +1,410 @@
+            r.installation_id,
+            (i.installation_id IS NOT NULL AND i.suspended_at IS NULL)
+                AS has_active_installation
+        FROM github_connected_repos r
+        LEFT JOIN github_installations i
+            ON i.installation_id = r.installation_id
+        WHERE r.user_id = %s AND r.repo_full_name = %s
+        LIMIT 1
+    """
+
+    with db_pool.get_admin_connection() as conn:
+        with conn.cursor() as cursor:
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)